Specs/Scene/PostProcessStageLibrarySpec.js

```diff
@@ -301,52 +305,41 @@ describe(
         new HeadingPitchRoll()
       );
 
-      const model = scene.primitives.add(
-        Model.fromGltf({
+      return loadAndZoomToModel(
+        {
           url: boxTexturedUrl,
+          incrementallyLoadTextures: false,
           modelMatrix: modelMatrix,
-          scale: 40.0,
-        })
-      );
-
-      let ready = false;
-      model.readyPromise.then(function () {
-        ready = true;
-      });
-
-      const offset = new Cartesian3(
-        -37.048378684557974,
-        -24.852967044804245,
-        4.352023653686047
-      );
-      scene.camera.lookAt(origin, offset);
+        },
+        scene
+      ).then(function () {
+        scene.camera.lookAt(origin, new HeadingPitchRange(0.0, 0.0, 1.0));
 
-      return pollToPromise(function () {
-        scene.render();
-        return ready;
-      }).then(function () {
+        // Render without depth of field
+        let originalColor;
         expect(scene).toRenderAndCall(function (rgba) {
+          originalColor = rgba;
           for (let i = 0; i < rgba.length; i += 4) {
             expect(rgba[i]).toBeGreaterThan(0);
-            expect(rgba[i + 1]).toEqual(0);
-            expect(rgba[i + 2]).toEqual(0);
+            expect(rgba[i + 1]).toBeGreaterThan(0);
```

Comment:
These lines were originally `toEqual(0)` because the old model was red, right?

Reply:
correct, I'm using a different model now.